dotCMS/src/main/java/com/dotcms/ai/app/AppKeys.java

```diff
@@ -52,6 +52,8 @@ public enum AppKeys {
     AI_MODELS_CACHE_TTL("com.dotcms.ai.models.supported.ttl", "28800"),
     AI_MODELS_CACHE_SIZE("com.dotcms.ai.models.supported.size", "64");
 
+    private static final String FALSE = "false";
```

Comment:
this may be promoted to a more generic class such as StringPool

Reply:
Done